content/spring-interview-questions/2-spring-aop.md

```diff
@@ -4,9 +4,12 @@
 --
 
 # What is the interviewer looking for?
-- One
-- Two
-- Three
+- What is aop ?
+- How is it used in your projects ?
+- What are the terminologies used in aop ?
+- What is Advice and explain any one advice ?
+- How do you enable the aop in spring ?
+- Can we handle transactional with aop ?
```

Comment:
Please remove any questions that don't have answers. We can add them when they are ready.